app/controllers/scim_rails/scim_users_controller.rb

```diff
@@ -82,13 +82,14 @@ def patch_update
       user = @company.public_send(ScimRails.config.scim_users_scope).find(params[:id])
 
       params["Operations"].each do |operation|
-        raise ScimRails::ExceptionHandler::UnsupportedPatchRequest if operation["op"] != "replace"
+        raise ScimRails::ExceptionHandler::UnsupportedPatchRequest unless ["Replace", "replace"].include?(operation["op"])
 
-        changed_attributes = permitted_params(operation["value"])
+        path_params = (operation.key?("path")) ? process_path(operation) : nil
```

Comment:
🌵 got some extra parentheses 
```suggestion
        path_params = operation.key?("path") ? process_path(operation) : nil
```